src/pages/my_resources/MyResources.tsx

```diff
@@ -41,58 +46,34 @@ const MyResources: FC = () => {
   return (
     <StyledPageContent>
       {loadingError && <ErrorBanner />}
-      <Typography variant="h2">{t('resource.unpublished_resources')}</Typography>
-      <List>
-        {isLoadingMyResources && <CircularProgress />}
-        {!isLoadingMyResources &&
-          resourcesUnpublished.length > 0 &&
-          resourcesUnpublished.map((resource: Resource, index: number) => (
-            <ListItem
-              button
-              component="a"
-              href={`/resource/${resource.identifier}`}
-              key={index}
-              alignItems="flex-start">
-              <ListItemIcon>
-                <ImageIcon />
-              </ListItemIcon>
-              <ListItemText
-                primary={`${resource.features.dlr_title} (${resource.features.dlr_content_type})`}
-                secondary={
-                  <Typography style={{ display: 'block' }} component="span" variant="body2" color="textPrimary">
-                    {resource.features.dlr_time_created}
-                  </Typography>
-                }
-              />
+      <div>
```

Comment:
denne diven trengs vel ikke.

Reply:
Uten den diven:
![image](https://user-images.githubusercontent.com/44363838/104454270-e5e6b900-55a5-11eb-926e-7b364029f7ad.png)
Med diven:
![image](https://user-images.githubusercontent.com/44363838/104454327-f8f98900-55a5-11eb-923b-3e00a78810a7.png)

Reply:
Den aligner de to listene slik at de starter på samme sted horisontalt.

Reply:
Men jeg kan kanskje rename den slik at det er tydelig at den har en misjon annet en å dele opp innhold unødvendig? Sette display: block, align-items: left? Noe i den duren?

Reply:
åja. så ikke at det var to StyledListWrapper under den.

Reply:
evt lage en tom styled component ` wrapper  = styled.div``;`   - men det er kanskje unødvendig

Reply:
Kanskje kalle den listMarginAlign med display block og align items? hvis den bare heter wrapper så uten noe styling tenker jeg det er fort gjort å fjerne den ved en senere anleding ved uhell.

Reply:
Da er det fikset